queries/eds/highlights.scm

```diff
@@ -0,0 +1,63 @@
+["="] @punctuation.delimeter
+
+(
+ (section_name) @variable.builtin
+ (#any-of? @variable.builtin "[FileInfo]" "[DeviceInfo]" "[DummyUsage]" "[MandatoryObjects]" "[OptionalObjects]")
+)
+
+(
+ (section_name) @variable.builtin
+ (#lua-match? @variable.builtin "^%[1")
+)
```

Comment:
Can we follow the style seen in pretty much every query? No dangling paren mainly..
```suggestion
((section_name) @variable.builtin
  (#lua-match? @variable.builtin "^%[1"))
```